python/src/opendp/extras/sklearn/pca.py

```diff
@@ -1,22 +1,35 @@
+'''
+.. TODO: Fix conflicts so we can get generated API docs for PCA.
+
+See our :ref:`tutorial on diffentially private PCA <dp-pca>`.
+'''
+
 from __future__ import annotations
-from opendp._extrinsics.make_np_pca import make_private_np_pca
+from opendp.extras.sklearn.make_pca import make_private_pca
 from opendp.mod import Measurement
 from opendp._lib import import_optional_dependency
 
 
 decomposition = import_optional_dependency('sklearn.decomposition', False)
 if decomposition is not None:
-    class SKLPCA(decomposition.PCA): # type: ignore[name-defined]
+    class _SKLPCA(decomposition.PCA): # type: ignore[name-defined]
+        '''
+        :meta private:
+        '''
         pass
 else: # pragma: no cover
-    class SKLPCA(object): # type: ignore[no-redef]
+    class _SKLPCA(object): # type: ignore[no-redef]
+        '''
+        :meta private:
+        '''
         def __init__(*args, **kwargs):
             raise ImportError(
                 "The optional install scikit-learn is required for this functionality"
             )
 
 
-class PCA(SKLPCA):
+class PCA(_SKLPCA):
+    # TODO: If I add a docstring here, it also tries to locate _SKLPCA, and fails
```

Comment:
This is the same issue as in LazyFrameQuery from Polars, where we have an API inheriting from an optional dependency. It's not clear to me which approach is better, but it would be best if both places ultimately solve the problem the same way.

Doesn't have to be resolved at this point.